src/vm/assembly.cpp

```diff
@@ -1462,6 +1462,62 @@ void ValidateMainMethod(MethodDesc * pFD, CorEntryPointType *pType)
     }
 }
 
+struct Param
```

Comment:
This is just extraction of part of the RunMain into a separate function to prevent mixing SEH and C++ EH in the same function.